Packs/CybleEventsV2/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### CybleEvents v2
+- Improved the fetching logic
```

Comment:
can you give a little explanation to the improvement done, something like:
Improved implementation of ***fetch-events*** command to/by .....